src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+          rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <span class="logo__container">
+          <a class="logo__link"
+             href="index.html">
+            <img class="logo__image"
+                 src="images/logo.png"
+                 alt="lo">
+          </a>
+        </span>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#"
+                 class="nav__item-link is-active">APPLE
```

Comment:
Tag is not closed here.

Also, check the requirements: `Uppercase letters for nav__list are made with styles, not hardcoded into html (you should have usual text with first uppercase letter in html)`, make sure to fix this